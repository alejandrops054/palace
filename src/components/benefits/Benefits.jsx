/* Sistema Intelligence 2.0
MÃ³dulo Contratos
VersiÃ³n 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import Nav from './NavBenefits'
import TablePromotions from './TablePromotions'
import Certificates from './Certificates'
import RefProgram from './ReferralProgram'
import DiscountProductsTable from '../discount-products/DiscountProductsTable'


const Benefits = () => {
    return (
        <form>
            <div className="row col-lg-12">
                <TablePromotions />
                <DiscountProductsTable/>
                <Certificates />
                <RefProgram />
            </div>
        </form>
    )
}

export default Benefits
