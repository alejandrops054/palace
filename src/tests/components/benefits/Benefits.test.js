/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import Benefits from '../../../components/benefits/Benefits'

describe('Benefits component',() => {
    test('Debe mostrar snapshot', () => {
        
        const wrapper = shallow(<Benefits/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})