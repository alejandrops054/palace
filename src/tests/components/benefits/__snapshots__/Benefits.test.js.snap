// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Benefits component Debe mostrar snapshot 1`] = `
<form>
  <div
    className="row col-lg-12"
  >
    <TablePromotions />
    <DiscountProductsTable />
    <Certificates />
    <RefProgram />
  </div>
</form>
`;
